### PR TITLE
adds custom message prefix

### LINKED ISSUE #95: add custom prefix for alert messages
We could add some custom prefix (in config) to the message eg:
[My Group bot for 49123] Freier Impfslot (11)! AstraZeneca/Vector

Dunno where to put in the config. `[ADVANCED] custom_message_prefix` maybe?

_Originally posted by @sibalzer in https://github.com/sibalzer/impfbot/issues/5#issuecomment-857458719_

### PR DESCRIPTION
**Pull request by paulypeter**
closes #95 

until we have a better idea where to put the setting

- [x] fix failing tests
- [x] find better regex

### CODE REVIEW

File: src/config_skeleton.py [hunk 1]
```diff
@@ -96,6 +96,11 @@
         },
     },
     "ADVANCED": {
+        "custom_message_prefix": {
+            "default": "Impfbot",
+            "type": str,
+            "regex": USER_AGENT_REGEX
```
Comment (paulypeter) [inline src/config_skeleton.py:102]: Needs a better regex

**Comment by paulypeter** [comment]
Phew.

File: src/config_skeleton.py (hunk 1 above)
Comment (paulypeter) [reply src/config_skeleton.py:102]: it's .* now

**Comment by paulypeter** [comment]
closes #95

**Review by sibalzer — APPROVED** [review]

**Pull request merged**